demo/src/main/java/com/smashingboxes/surelockdemo/LoginActivity.java

```diff
@@ -152,7 +152,7 @@ private void handleLoginClick() {
         String username = usernameView.getText().toString();
         String password = passwordView.getText().toString();
 
-        if (fingerprintCheckbox.isChecked() && surelock.fingerprintAuthIsSetUp(true)) {
+        if (fingerprintCheckbox.isChecked() && Surelock.fingerprintAuthIsSetUp(true, this)) {
```

Comment:
This is super minor, but typically Context reference parameters are supposed to be passed in as the first parameter. Mind changing this real quick? 😬